src/tests/test_load_csv.py

```diff
@@ -61,7 +65,10 @@ def test_multiple_columns_embedding(mocker, dbsession, csv_file: str):
     )
 
     mock_generate_embeddings.assert_called_with(
-        ["cat1\nsubcat1\ncontent1", "cat2\nsubcat2\ncontent2"],
+        [
+            "category: cat1\nsubcategory: subcat1\ncontent: content1", 
+            "category: cat2\nsubcategory: subcat2\ncontent: content2"
+        ],
```

Comment:
this is the default way that langchain's CSVLoader join columns that are not metadata. This makes sense (for LLM tasks) since each document will have prefixes that explain their values, and the actual formatting that documents will assume on retrieval is supposed to be customized as well, but I didn't see any negative impact in practice (current examples are [here](https://github.com/talkdai/dialog/blob/main/src/dialog/llm/agents/default.py#L53) and  even [better here](https://github.com/talkdai/dialog/blob/main/src/dialog/llm/agents/lcel.py#L60-L61), where the formatter is a runnable lambda function; custom plugins can deal with formatting as they wish)

Reply:
Ok, is there any necessary change over here or just a quick explanation?

Reply:
no, there is not necessary change here, but users using our classes should notice this change when looking at the the context formatting, but without regressions.